README.md

```diff
@@ -1 +1,34 @@
-# Operator Marketplace
+# Marketplace Operator
+
+## Project Status: pre-alpha
+The project is currently pre-alpha and it is expected that breaking changes to the API will be made in the upcoming releases.
+
+## Prerequisites
+In order to deploy the Marketplace Operator, you must:
+1. Have an OpenShift cluster with Operator Lifecycle Manager [installed](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md).
```

Comment:
Please say "OKD or Kubernetes cluster"

Reply:
Removed all references to `OpenShift` with `OKD`.